pkg/console/cookie_auth.go

```diff
@@ -45,27 +45,22 @@ type authCookie struct {
 	Expiry       time.Time
 }
 
-// setCookie sets the authCookie on the request.
-func (console *Console) setAuthCookie(c echo.Context, value authCookie) error {
-	return console.AuthCookie().Set(c, value)
-}
-
-// getCookie returns the authCookie from the echo context.
 func (console *Console) getAuthCookie(c echo.Context) (authCookie, error) {
 	value := authCookie{}
 	ok, err := console.AuthCookie().Get(c, &value)
 	if err != nil {
 		return authCookie{}, err
 	}
-
 	if !ok {
-		return authCookie{}, echo.NewHTTPError(http.StatusUnauthorized, "You are not logged in")
+		return authCookie{}, echo.NewHTTPError(http.StatusBadRequest, "No auth cookie")
```

Comment:
Should this return `401` instead of `400`? 
This rollbacks changes made in https://github.com/TheThingsNetwork/lorawan-stack/pull/1007

Reply:
Ok

Reply:
Done.